server/src/main/java/org/elasticsearch/index/mapper/MappingLookup.java

```diff
@@ -284,15 +276,6 @@ public Iterable<Mapper> fieldMappers() {
         return fieldMappers.values();
     }
 
-    /**
-     * Gets the postings format for a particular field
-     * @param field the field to retrieve a postings format for
-     * @return the postings format for the field, or {@code null} if the default format should be used
-     */
-    public PostingsFormat getPostingsFormat(String field) {
-        return completionFields.contains(field) ? CompletionFieldMapper.postingsFormat() : null;
-    }
```

Comment:
This is no longer used anywhere now that `CompletionFieldMapper::postingsFormat` is called directly (see comments above).